op-scim-bridge/chart/op-scim-bridge-mp/Chart.yaml

```diff
@@ -15,12 +15,12 @@ icon: https://1password.com/img/logo-v1.svg
 sources:
   - https://app-updates.agilebits.com/product_history/SCIM
 # appVersion should match the value of publishedVersion in the schema.yaml
-appVersion: "2.8.4"
+appVersion: "2.9.0"
 dependencies:
   - name: op-scim-bridge
     version: ~2
     repository: https://1password.github.io/op-scim-helm/
     condition: redis.enabled
   - name: common
-    version: ~1
+    version: ~2
```

Comment:
There's a major version upgrade here for the bitnami chart. I'm looking into if there are any compatibility issues for our chart.

Additionally `helm dependency update .` is yet to be committed to this branch for this bitnami common chart. `charts/common-2.14.1.tgz`

Reply:
After updating the dependencies in this chart, I set up a k8s cluster in gcp and successfully ran through all of the steps here: https://github.com/kubernetes-sigs/application which included `make app/build`, `mpdev install`, and `mpdev verify`. I'll commit and push the changes to this branch, then we can attempt to deploy again tomorrow.